.ci/scripts/validate_commit_message.py

```diff
@@ -6,14 +7,34 @@
 from github import Github
 
 KEYWORDS = ["fixes", "closes"]
+BLOCKING_REGEX = [
+    "DRAFT",
+    "WIP",
+    "NOMERGE",
+    r"DO\s*NOT\s*MERGE",
+    "EXPERIMENT",
+]
 NO_ISSUE = "[noissue]"
-CHANGELOG_EXTS = [".feature", ".bugfix", ".doc", ".removal", ".misc", ".deprecation"]
+# TODO (On a rainy afternoon) Fetch the extensions from pyproject.toml
+CHANGELOG_EXTS = [
+    ".feature",
+    ".bugfix",
+    ".doc",
+    ".removal",
+    ".misc",
+    ".deprecation",
+    ".translation",
+    ".devel",
+]
 
 sha = sys.argv[1]
-project = "pulp-cli-deb"
+project = "pulp-cli"
```

Comment:
This sounds wrong. But is it even used anywhere?

Reply:
Yea I noticed that as well. didn't see it in the diff before the commit. I don't think that it's used somewhere I just copied it from `pulp-cli`. We can remove it and try it out.

Reply:
I would change it back to "pulp-cli-deb" even if we are not sure what it is used for. Changing it from `pulp-cli-deb` to `pulp-cli` just strikes me as wrong.

Reply:
I'd opt for removing it.

Reply:
Removing it is also fine.

Reply:
Removed, resolving!